src/devtools/client/debugger/src/components/SecondaryPanes/Breakpoints/BreakpointHeading.tsx

```diff
@@ -60,7 +84,8 @@ class BreakpointHeading extends PureComponent {
         <Redacted className="breakpoint-heading-label">{fileName}</Redacted>
         <div className="breakpoint-heading-actions">
           <CloseButton
-            handleClick={e => this.removeBreakpoint(e)}
+            buttonClass={null}
```

Comment:
:speech_balloon: `<CloseButton> just checks truthiness.  The file is still JS, and TS was whining I wasn't passing in all props.